deep_qa/layers/wrappers/encoder_wrapper.py

```diff
@@ -23,3 +23,4 @@ def compute_mask(self, x, input_mask=None):
         else:
             # An output bit is 0 only if the  bits corresponding to all input words are 0.
             return K.any(input_mask, axis=-1)
+        
```

Comment:
You can just get `git checkout master deep_qa/layers/wrappers/encoder_wrapper.py` to revert this.  No need to have this here.